src/platform-integration/android/platform-views.md

```diff
@@ -406,6 +406,10 @@ android {
 }
 ```
 
+#### Multi threaded draw calls
+
+Android's PlatformView implementation requires that the view is manually invalidated when the Android View draws using a separate thread. 
```

Comment:
**issue:** Needs semantic line breaks. sembr.org

Needs to follow confition, then action.

```markdown
When the Android View draws using a separate thread,
you must invalidate the view according to 
Android's PlatformView implementation. 
```